packages/app/cypress/integration/components/search/SearchPage.js

```diff
@@ -41,15 +56,29 @@ describe('SearchPage', () => {
             });
         },
       });
-      cy.contains('Search');
 
-      cy.contains(results[0].document.title);
-      cy.contains(results[0].document.text);
-      cy.get(`a[href="${results[0].document.location}"]`).should('be.visible');
+      cy.expect(Heading('Search').exists());
+
+      cy.expect(
+        List()
+          .find(
+            ListItem(
+              and(
+                including(results[0].document.title),
+                including(results[0].document.text),
```

Comment:
I'd leave out the `List().find()` since it doesn't add anything (if List() is not unique, then it will raise an error) 

And maybe even not use a matcher.

```js
cy.expect([
  ListItem(including('backstage')).exists(),
  ListItem(including("Backstage system documentation")).exists()
]);